docs/overview.md

```diff
@@ -118,6 +121,26 @@ Stargz snapshotter doesn't share credentials with containerd so credentials spec
 # ctr-remote image rpull --user <username>:<password> docker.io/<your-repository>/ubuntu:18.04
 ```
 
+#### CRI-based authentication
+
+Following configuration enables stargz snapshotter to pull private images on Kubernetes.
+The snapshotter works as a proxy of CRI Image Service and exposes CRI Image Service API on the snapshotter's unix socket (i.e. `/run/containerd-stargz-grpc/containerd-stargz-grpc.sock`).
+The snapshotter acquires registry creds by scanning requests.
+
+You must specify `--image-service-endpoint=unix:///run/containerd-stargz-grpc/containerd-stargz-grpc.sock` option to kubelet.
+This is the recommended way to enable lazy pulling of private images on Kubernetes.
```

Comment:
I’m feeling kubeconfig-based auth is more preferable for practitioners, as they wouldn’t want to modify kubelet args.

Reply:
Also, depending on the deployment tool, you can’t specify the image service socket that is different from the runtime socket. (Does stargz snapshotter also proxify the runtime service?)

Reply:
Thank you for the comment. Fixed it.